libs/platform/src/lib/components/upload-collection/dialogs/move-to/move-to.component.ts

```diff
@@ -0,0 +1,124 @@
+import { Component, Inject, OnInit } from '@angular/core';
+
+import { DIALOG_REF, DialogRef, DialogService, DialogConfig } from '@fundamental-ngx/core';
+import { take } from 'rxjs/operators';
+
+import { UploadCollectionItem, UploadCollectionFolder } from '../../models/upload-collection.models';
+import { NewFolderComponent } from '../new-folder';
+
+function isObject<T>(item: T): boolean {
+    return typeof item === 'object' && !Array.isArray(item) && item !== null;
```

Comment:
What's happen with `undefined`?
What do you think about this?
`return !!item && typeof item === 'object';`
or
`return !!item && item.contructor === ({}).constructor;`

Reply:
if we call `isObject(undefined)` resault will be `false` as expected
the function `isObject` must define only object (`{}`), not an array and other

Reply:
Maybe let's put this method somewhere into utils?